senior-backend-engineer/restful-api-categories.md

```diff
@@ -16,20 +16,20 @@ The category entity has the following structure:
 ## Required functionality:
 
 * Clients can create a new category.
-* Clients can retrieve a category by id or slug.
+* Clients can retrieve a category by id or key.
 * Clients can retrieve a tree of children under a specific category.
 * Clients can update the visibility status of a specific category.
```

Comment:
Does it make sense to rearrange the category parent to be added as an additional requirement? Shifting a whole subtree? What do the requirements from a proper category management look like?